manifests/system.pp

```diff
@@ -76,8 +84,10 @@
   # the fact won't work until rvm is installed before puppet starts
   if $facts['rvm_version'] and !empty($facts['rvm_version']) {
     if ($version != undef) and ($version != present) and ($version != $facts['rvm_version']) {
-      if defined(Class['rvm::gnupg_key']) {
-        Class['rvm::gnupg_key'] -> Exec['system-rvm-get']
+      if $signing_keys {
+        $signing_keys.each |Hash[String[1], String[1]] $key| {
```

Comment:
typo?
```suggestion
        $signing_keys.each |Hash[String[1], String[1] $key| {
```

Reply:
Looks good to me `Hash[ String[1], String[1] ]`

Reply:
ahh that's the closing bracket from the Hash type